interfaces/interfaces.tsx

```diff
@@ -1,8 +1,7 @@
 export interface Member {
   name: string;
-  id: number;
   skill: string;
-  language: string;
+  language: string[];
   gender: string;
```

Comment:
if `language` is an array now, you should rename it to `languages`

Reply:
ok